sssd_test_framework/hosts/client.py

```diff
@@ -145,6 +145,7 @@ def restore(self, backup_data: Any | None) -> None:
                 rm --force --recursive "$2"
                 if [ -d "$1" ] || [ -f "$1" ]; then
                     cp --force --archive "$1" "$2"
+                    restorecon -rf "$2"
```

Comment:
```
       -f infilename
              infilename contains a list of files to be processed. Use “-” for stdin.
```
You probably want just `restorecon -r "$2"`

Reply:
I was not aware --archive  knows also selinux contexts.

Reply:
I will close this one and focus on the mh one as it is probably the root cause of the failures. (this was more like a bandaid)